test/testdata/parser/whitequark/parse_send_call_1.rb.parse-tree.exp

```diff
@@ -12,7 +12,7 @@ Begin {
         args = [
         ]
       }
-      method = <U !>
+      method = <U call>
```

Comment:
What do you think about adding the test case from #1029 ?

Reply:
I had originally added more tests, but then found that there were already two tests in the whitequark suite. Given that they covered that case, I decided to leave it out.